api/graphql/tests/test_reservation_unit_image.py

```diff
@@ -15,12 +15,12 @@
     ReservationUnitImageFactory,
 )
 
-DEFAULT_GRAPHQL_URL = "/graphql/"
-
 
 class ReservationUnitImageCreateTestCase(
     GrapheneTestCaseBase, GraphQLFileUploadTestCase
 ):
+    GRAPHQL_URL = "/graphql/images/"
```

Comment:
Looks like this could be in `GrapheneTestCaseBase` just to avoid code duplication in each TestCase class 🤔

Reply:
It actually applies only to the reservationUnitImage gql since they're now in their own url so applying in the base level wouldn't do.